django/contrib/gis/db/backends/mysql/operations.py

```diff
@@ -23,6 +23,10 @@ def is_mysql_5_6(self):
         return self.connection.mysql_version < (5, 7, 6)
 
     @cached_property
+    def returns_invalid_geometry_collection(self):
```

Comment:
Some comments about this would be helpful. Is it a bug or is it expected MySQL behavior that will be needed for the foreseeable future?